extensions/cornerstone/src/hps/mammo.ts

```diff
@@ -0,0 +1,57 @@
+export const mgCC = {
+  id: 'mgCC',
+  name: 'mammo-cc',
+  imageLoadStrategy: 'interleaveTopToBottom',
```

Comment:
Leave the image load strategy out now - it is better handled with the progressive loading.